Rect bbox
int32 id 
Location3D location
